src/ts/ContentMetadataBuilder.ts

```diff
@@ -121,11 +129,11 @@ export class ContentMetadataBuilder {
     this.metadata.applicationName = newValue;
   }
 
-  set custom(newValue: CustomContentMetadata) {
+  set custom(newValue: Metadata['custom']) {
```

Comment:
Didn't know you can use types like that 😲 Thanks for showing that to me!

Reply:
Yeah, it's weird. I commented on it in the sample https://github.com/bitmovin/bitmovin-player-web-samples/blob/main/react/bundled-player/package-based-with-conviva/src/App.tsx#L77.

I've just added a comment in this repo as well - https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/106/commits/c560691e614fdd9a01041493253c90ad67d95b83.

Reply:
And here is one more comment https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/106/commits/06cd7394d6fe438d8208460283162123d847bf9f.